.github/workflows/deploy_production.yml

```diff
@@ -34,24 +34,14 @@ jobs:
           ghcr.io/zooniverse/subject-set-search-api:${{ github.sha }}
         build-args: |
           BUILD_DATE: ${{ steps.date.outputs.BUILD_DATE }}
-  deploy:
-    runs-on: ubuntu-latest
-    needs: build_and_push_image
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v3
-
-    - uses: azure/login@v1
-      with:
-        creds: ${{ secrets.AZURE_AKS }}
-
-    - name: Set the target AKS cluster
-      uses: Azure/aks-set-context@v3
-      with:
-        cluster-name: microservices
-        resource-group: kubernetes
 
-    - name: Modify & apply template
-      run: |
-        kubectl apply -f kubernetes/deployment-production.yaml
-        kubectl rollout restart deployment subject-set-search-api-production-app
```

Comment:
The shared workflow doesn't run this restart. Do we need it?

Reply:
I think this might ensure that we restart even when the latest commit hasn't changed (ie. when new subject set data is added.) I'm not sure.